MachObfuscatorTests/SymbolManglingTests/ExportTrieMangler_Tests.swift

```diff
@@ -0,0 +1,72 @@
+import XCTest
+
+class ExportTrieMangler_Tests: XCTestCase {
+
+    var sut: ExportTrieMangler!
+
+    var rootTrie: Trie!
+
+    override func setUp() {
+        sut = ExportTrieMangler()
+        var childrenLayer0 = trieChildren(number: 3)
+        var childrenLayer1 = trieChildren(number: 2)
+        var childrenLayer2 = trieChildren(number: 2)
+        let childrenLayer3 = trieChildren(number: 2)
+
+        childrenLayer2 = assignChildren(childrenLayer3, to: childrenLayer2)
+        childrenLayer1 = assignChildren(childrenLayer2, to: childrenLayer1)
+        childrenLayer0 = assignChildren(childrenLayer1, to: childrenLayer0)
+
+        rootTrie = Trie(exportsSymbol: false,
+                            labelRange: 0..<UInt64(2),
```

Comment:
This is probably not a case, but real-world root trie node will always have empty label.
```suggestion
                            labelRange: 0..<0,
```

Reply:
Sure, same as above 👍 :)